### PR TITLE
reorganize release note 

### PR DESCRIPTION
**Pull request by mirnawong1**
reorganize the order of the release note from '07-aug-2022' to '06-aug-2022' since i added a july release note and that needs to be '07'

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

File: website/docs/docs/dbt-cloud/release-notes/06-Aug-2022/ide-improvement-beta.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ sidebar_label: "Enhancement: New cloud IDE beta"
 tags: [Aug-16-2022]
 ---
 
+:::info Beta Feature 
+Read more about the [Cloud IDE beta](https://www.getdbt.com/blog/staging-highlights-the-latest-from-dbt-labs/) and [submit your expression of interest](https://docs.google.com/forms/d/e/1FAIpQLSdlU65gqTZPyGAUc16SkxqTc50NO9vdq_KGx1Mjm_4FB_97FA/viewform) to join the new Cloud IDE Beta group.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/release-notes/06-Aug-2022/ide-improvement-beta.md]: I think we can standardize on lowercase "b" in beta unless it's the first word in a title. 
```suggestion
Read more about the [Cloud IDE beta](https://www.getdbt.com/blog/staging-highlights-the-latest-from-dbt-labs/) and [submit your expression of interest](https://docs.google.com/forms/d/e/1FAIpQLSdlU65gqTZPyGAUc16SkxqTc50NO9vdq_KGx1Mjm_4FB_97FA/viewform) to join the new Cloud IDE beta group.
```

File: website/docs/docs/dbt-cloud/release-notes/06-Aug-2022/ide-improvement-beta.md [hunk 2]
```diff
@@ -6,6 +6,11 @@ sidebar_label: "Enhancement: New cloud IDE beta"
 tags: [Aug-16-2022]
 ---
 
+:::info Beta Feature
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/release-notes/06-Aug-2022/ide-improvement-beta.md]: I think we can use a standard sentence case here
```suggestion
:::info Beta feature 
```

**Review by runleonarun — APPROVED** [review]
Approved so you can merge once you consider my small suggestions!

**Comment by mirnawong1** [comment]
thank you @runleonarun for the suggestion and you're absolutely right about the consistency

**Pull request merged**